### DISCUSSION TITLE
Support embedding PWAs

### DISCUSSION THREAD

**Discussion by raayu83** [tauri-apps/tauri#8454] [Q&A]
Does Tauri support embedding PWAs?
E.g. for creating something like a Tauri based Teams Client for Linux. (which is only available as PWA for LInux)

**Comment by FabianLars** [comment]
Hi, sorry for the delay. I'm not sure if i may be misunderstanding the question but PWAs are really just normal websites and Tauri can display remote websites just fine.

If by embedding you mean that it should only show in a part of the window and not fully fill it you'll have to use iframe in v1 (if the website allows that) or wait for multi webview support in v2.
